Tasks/AppCenterDistributeV3/azure-blob-upload-helper.ts

```diff
@@ -0,0 +1,50 @@
+import * as AzureStorage from "azure-storage";
+import * as Url from "url";
+
+import { inspect } from "util";
+
+export class AzureBlobUploadHelper {
```

Comment:
Do we have tests for this?

Reply:
We don't at the moment. This code was borrowed from the App Center CLI and we didn't find any matching tests. We mostly weren't sure how they would fit into the rest of the test suite. But we can try to retrofit if it's important.

Reply:
Ideally we should have tests for these? If it's significant amount of effort, we probably have to skip. I would suggest investigate what it takes though.

Reply:
Given that it's basically some glue code wrapping the Azure Storage SDK, we're going to skip adding tests for now as we don't think the ROI is worth the effort.